pset-iterators.js

```diff
@@ -87,7 +95,12 @@ Tip: your new array should contain 45 elements
 console.log('Problem 2:')
 
 // Add your code below this line
+const presidentialParties = []
```

Comment:
@kertcodes nice use of `.forEach()` an alternative approach would have been to use `.map()`